### PR TITLE
Added merge_soreted_list func and its pytest

### PR DESCRIPTION
**Pull request by aruaaa17**
Could you please review my code?

### CODE REVIEW

File: tests/merge_sorted_lists_test.py [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+    l2 = [1]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2]
+
+
+def test_for_1_2_3_and_4_5_6():
```
Comment (jping0220) [inline tests/merge_sorted_lists_test.py]: can you rename this function name more readable? thanks

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
+
+def test_for_1_and_2():
+    # Arrange
+    l1 = [1]
+    l2 = [2]
```
Comment (sophiat8832) [inline tests/merge_sorted_lists_test.py]: Can you make l1 and l2 more descriptive variables?

**Review by sophiat8832 — CHANGES_REQUESTED** [review]
Please look at my comment. PLEASE DO IT!!!

**Review by jping0220 — COMMENTED** [review]
just one review for you

**Comment by aruaaa17** [comment]
Thank you Thank you ☃︎

**Pull request merged**